.github/workflows/ci_cd.yml

```
@@ -76,43 +76,7 @@ jobs:
         env:
           CHANGES: ${{ needs.get-changes.outputs.changes }}
         shell: python
```

Since you're now invoking a Python script using `python <script.py>`, the shell must be changed to Bash (or dropped). This also applies to the other places with a similar change.

```suggestion
        shell: bash
```

or

```suggestion
```

---

Sure, I have made the specified changes in the latest commit

---

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/ci_cd.yml#L73-L81

---

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/ci_cd.yml#L73-L81

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/ci_cd.yml#L795-L805

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/deploy-production-api.yml#L61-L64

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/deploy-production-nuxt.yml#L61-L64

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/deploy-staging-api.yml#L61-L64

https://github.com/rahulbollisetty/openverse/blob/bc7bf3d40a7263a37230bde46356841a5bc3d85e/.github/workflows/deploy-staging-nuxt.yml#L61-L64

---

@dhruvkb I have made the suggested changes. Please have a look, and let me know if there are any more issues.